### ISSUE TITLE
Landscape orientation - menu closing

### ISSUE THREAD

**Issue by JuliannaR** [cds-snc/covid-alert-app#769]
Here's on another landscape issue in iOS iPhone XR 13.5.1. When in landscape view on the "COVID Alert is off" page, I'm unable to move the menu away to read the text on the page as in the screenshots.

![IMG_1353 (1)](https://user-images.githubusercontent.com/35537308/87828658-6886f980-c84b-11ea-9a26-2321f6e9a3af.PNG)
![IMG_1353](https://user-images.githubusercontent.com/35537308/87828661-6a50bd00-c84b-11ea-9594-8db5bd9ff3c4.PNG)
![IMG_1354 (1)](https://user-images.githubusercontent.com/35537308/87828670-6c1a8080-c84b-11ea-86a7-2177b074117e.PNG)
![IMG_1354](https://user-images.githubusercontent.com/35537308/87828672-6d4bad80-c84b-11ea-9d0f-15048a9d47e0.PNG)

**Comment by JuliannaR** [comment]
@amazingphilippe we will need to look at how to best address some of the landscape orientation issues going forward. I'll bring this to standup.

**Comment by amazingphilippe** [comment]
I think we should reorder the flex layout so that it takes less vertical space on a landscape oriented devices. Especially with that gray button.

**Comment by timarney** [comment]
>  I'm unable to move the menu away to read the text on the page as in the screenshots.

Is the content not scrollable?

**Comment by JuliannaR** [comment]
@timarney the target area in which you can scroll seems really small so depending on a device resolution its challenging to do this because the size of the shield is so large in landscape view.

**Comment by amazingphilippe** [comment]
This issue challenges the idea of the shield in landscape mode. Which is a good thing, because its not a great experience right now. We can make it better, but the implications of changing this behavior are quite large.

1. Should we consider an easy fix? Like removing the symbol or the app name in landscape only.
2. A harder fix. Which implies more of a redesign of how we display a permanent overlay ("the shield") in portrait and landscape, but in a way that works well for both.

**Comment by JuliannaR** [comment]
@emilykuret this is something we will need to triage to set aside time to work on.

**Comment by emilykuret** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Explore restructuring the menu](https://trello.com/c/wlYxBsAd/293-explore-restructuring-the-menu)

**Comment by timarney** [comment]
Closing this will re-open once the card gets some activity.
